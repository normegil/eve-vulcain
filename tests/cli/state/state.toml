bin.name = "eve-vulcain"
args = "state -vvv"
status.code = 0
