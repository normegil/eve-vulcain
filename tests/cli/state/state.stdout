EveVulcain Test User
Wallet: 123,456,789.12 ISK
Location: The Forge > Kimotoro > Maurasi (0.9) > Maurasi - BPs Center, read BIO
Corporation: School of Applied Knowledge

Jobs:
	                  Drone Link Augmentor I        56 run(s)          [..]


Market orders:
	                    Antipharmakon Aeolis         1/1                  20 ISK/u


